tests/unit/mongo.spec.ts

```diff
@@ -37,6 +46,24 @@ describe('Pipeline to mongo translator', () => {
     ]);
   });
 
+  it('can generate select steps with escaped characters', () => {
+    const pipeline: Pipeline = [
+      { name: 'domain', domain: 'test_cube' },
+      { name: 'select', columns: ['Man$', 'Re$$gion'] },
+    ];
+    const querySteps = mongo36translator.translate(pipeline);
+    expect(querySteps).toEqual([
+      { $match: { domain: 'test_cube' } },
+      {
+        $project: {
+          Man__DOLLAR_SIGN__: 1,
+          Re__DOLLAR_SIGN____DOLLAR_SIGN__gion: 1,
+        },
+      },
+      { $project: { _id: 0 } },
+    ]);
+  });
```

Comment:
Testing this feature is gonna cause quite a duplication in tests. Does that feel OK to everyone?
It seems to be that it's the only option to test if every field name is correctly escpaed, without changing the actual tests too much.